test/metatx/ERC2771Context.test.js

```diff
@@ -12,8 +12,10 @@ const ContextMockCaller = artifacts.require('ContextMockCaller');
 const { shouldBehaveLikeRegularContext } = require('../utils/Context.behavior');
 
 contract('ERC2771Context', function (accounts) {
+  const MAX_UINT48 = new BN('2').pow(new BN('48')).sub(new BN('1')).toString();
```

Comment:
IMO that is more readable (no need for all the `new BN` 
```suggestion
  const MAX_UINT48 = web3.utils.toBN(1).shln(48).subn(1).toString();
```

Reply:
Ahh, perfect, thanks!